systemtest/src/test/java/io/strimzi/systemtest/AbstractST.java

```diff
@@ -110,7 +110,7 @@ public abstract class AbstractST implements TestSeparator {
 
     public static Random rng = new Random();
 
-    public static final int MESSAGE_COUNT = 100;
+    public static final int MESSAGE_COUNT = Constants.MESSAGE_COUNT;
```

Comment:
You can remove such value from `AbstractST`, and only use it in appropriate places (i.e., as you do in HttpBridgeKafkaExternalListenerST`).

Reply:
I saw this too, but removal of `MESSAGE_COUNT` from `AbstractST` would end in more refactoring, which is for different PR FMPOV.

Reply:
Okay 👍